### PR TITLE
user event page

### PR DESCRIPTION
**Pull request by jowlee**

### CODE REVIEW

**Comment by samanthajlee2** [comment]
"user event page" seems vague...

good that events are not json anymore

events should be clickable right? (would I want to make clickable events it's own issue? and merge this branch?)

**Comment by seveibar** [comment]
@leej42

File: client/app/student/events/studentevents.html [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
     </div>
     <div class='col-sm-9'>
-      <div class='event-page events'>
+      <div class='event-page student-events'>
         <div class='event' ng-repeat='event in user.events'>
           <div class='title'>{{event.info.title}}</div>
-          <div class='notes'>{{event.additionalNotes}}</div>
-          <div class='times'>{{event.info.times}}</div>
+          <div class='notes'>Note: {{event.additionalNotes}}</div>
+          <!-- <div class='times'>{{event.info.times}}</div> -->
```
Comment (kmcnellis) [inline client/app/student/events/studentevents.html:14]: Ghost code?

File: client/app/student/events/studentevents.html [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
       <sidebar/>
     </div>
     <div class='col-sm-9'>
-      <div class='event-page events'>
+      <div class='event-page student-events'>
         <div class='event' ng-repeat='event in user.events'>
           <div class='title'>{{event.info.title}}</div>
-          <div class='notes'>{{event.additionalNotes}}</div>
-          <div class='times'>{{event.info.times}}</div>
+          <div class='notes'>Note: {{event.additionalNotes}}</div>
```
Comment (kmcnellis) [inline client/app/student/events/studentevents.html:12]: Why have the "Note: " ? It feels odd

**Comment by kmcnellis** [comment]
Its a good start - we should file an issue to add the pictures and times, though. (And a clickable link to the submissions page with the event pre-populated)

**Comment by seveibar** [comment]
#featurecreep haha @leej42 you should fix the ghost code so we can merge

**Comment by seveibar** [comment]
I added a comment to the ghost code- which I think is good

**Pull request merged**